softlayer/common/softlayer_helper.go

```diff
@@ -14,10 +14,11 @@ import (
 )
 
 var (
-	TIMEOUT             time.Duration
-	POLLING_INTERVAL    time.Duration
-	LocalDiskFlagNotSet bool
-	LengthOfHostName    int
+	TIMEOUT                   time.Duration
+	POLLING_INTERVAL          time.Duration
+	LocalDiskFlagNotSet       bool
```

Comment:
I am uncomfortable why these globals exist. I can understand if they need to be set. But the `LocalDNSConfigurationFile` is always `/etc/hosts` why not make it a constant? Is there a case where it will be different?